js/data.js

```diff
@@ -12,6 +12,17 @@ var data = {
   entries: []
 };
 
+var entryData = {
```

Comment:
Rather than make a new data model like this, you should make use of the `entries` property on the existing data model 👍